packages/modal-ui-js/project.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/modal-ui-js/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nrwl/js:tsc",
```

Comment:
Would it be possible to change the builder to `rollup`?
Make it the same as in other wallets this way the bundle will be either in ESM or CJS otherwise, we can run into issues like we had before the package might not work on SSR frameworks.